core-services/release-controller/_releases/release-ocp-4.6-ppc64le.json

```diff
@@ -13,5 +13,9 @@
     "tag":{"tagRef":{"name":"4.6"}}
   },
   "verify":{
+    "ppc64le":{
+      "optional":true,
+      "prowJob":{"name":"periodic-ci-openshift-multiarch-master-nightly-4.6-ocp-installer-remote-libvirt-ppc64le"}
```

Comment:
All the jobs name were updated 
`pull-ci-openshift-multiarch-master-nightly-4.6-ocp-installer-remote-libvirt-ppc64le`

Reply:
changed them to use periodic-***